customer/structures.go

```diff
@@ -36,3 +41,31 @@ type URLInfo struct {
 	ImageWidth       int    `json:"image_width"`
 	ImageHeight      int    `json:"image_height"`
 }
+
+type DynamicConfiguration struct {
+	GroupID             int    `json:"group_id"`
+	ClientLimitExceeded bool   `json:"client_limit_exceeded"`
+	DomainAllowed       bool   `json:"domain_allowed"`
+	ConfigVersion       string `json:"config_version"`
+	LocalizationVersion string `json:"localization_version"`
+	Language            string `json:"language"`
+}
+
+type ConfigButton struct {
+	ID           string `json:"id"`
+	Type         string `json:"type"`
+	OnlineValue  string `json:"online_value"`
+	OfflineValue string `json:"offline_value"`
+}
+
+type StaticConfiguration struct {
+	Buttons        []ConfigButton               `json:"buttons"`
+	TicketForm     *Form                        `json:"ticket_form,omitempty"`
+	PrechatForm    *Form                        `json:"prechat_form,omitempty"`
+	AllowedDomains []string                     `json:"allowed_domains,omitempty"` // CHECK: why not in docs?
```

Comment:
So, did you check it? ;)

Reply:
Created a task for adding allowed_domains to docs